SERVICES.md

```diff
@@ -0,0 +1,457 @@
+# Azure services and package import paths
```

Comment:
Is this file necessary? The `godoc` documentation covers all services in a table and is substantially less fragile since it is automatically updated.

Reply:
for now I'd like to include it as we need to hold users' hands through the transition from `arm/` to `services/` and there are a lot of services+versions permutations. i think it's helpful to be upfront about that. let's consider moving or relying on something else over time though, thanks!